### ISSUE TITLE
tf::transformTFToEigen() refuses to take Eigen::Isometry3d as output parameter

### ISSUE THREAD

**Issue by acschaefer** [ros/geometry#118]
When I do this:

```
tf::Transform t;
Eigen::Isometry3d e;
tf::transformTFToEigen(t, e);
```

gcc tells me:

```
error: invalid initialization of reference of type Eigen::Affine3d& 
{aka Eigen::Transform<double, 3, 2>&}
from expression of type Eigen::Isometry3d
{aka Eigen::Transform<double, 3, 1>} 
         tf::transformTFToEigen(t, e);
                                    ^
```

It seems tf has trouble recognizing the right Eigen data type?

**Comment by spaghetti-** [comment]
Compiles fine for me. Check if you have #113 pulled and you are calling `find_package` on `tf_conversions`.

**Comment by tfoote** [comment]
Closing as this looks to have been resolved in #113
